src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -1709,7 +1721,7 @@ def twists(self):
 
             sage: # needs sage.rings.finite_rings
             sage: K = GF(3**5)
-            sage: [E.ainvs() for E in EllipticCurve(j=K(1)).twists()]
+            sage: [E.ainvs() for E in EllipticCurve(j=K(1)).twists()] # random
```

Comment:
If this is just for the ordering, use `sorted()` instead of `# random`. If the curves are actually different, it would be good to add a check that the returned list has the correct length and that the curves all have the correct 𝑗‑invariant and that they are pairwise non-isomorphic over the base field.